ships/0021-local-source-upload.md

```diff
@@ -0,0 +1,350 @@
+---
+title: local-source-upload
+authors:
+  - @otaviof
+reviewers:
+  - @alicerum
+  - @adambkaplan
+  - @gabemontero
+  - @qu1queee
+  - @SaschaSchwarze0
+approvers:
```

Comment:
Please add me and @SaschaSchwarze0 as approvers.

Reply:
Please consider the latest changes.